nodebuilder/p2p/bootstrap.go

```diff
@@ -1,10 +1,22 @@
 package p2p
 
 import (
+	"os"
+
 	"github.com/libp2p/go-libp2p/core/peer"
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+var IsBootstrapper = getEnv("CELESTIA_BOOTSTRAPPER", "false") == "true"
```

Comment:
I think it would be more go-way if `IsBootstrapper` was a function and it would populate `Config` struct field during struct creation. Global var are usually avoided due to ability to be overwrited from elsewhere.

Reply:
We want to read from env var can be overwritten anytime

Reply:
But we need to read env only in nodebuilder, meaning it will be called only on launch. There is no need to store it in var after node has started.

Reply:
That's fine I you feel strongly about it.

Reply:
@smuu, pls apply the suggestion